openapi/auth-server.yaml

```
@@ -10,7 +10,7 @@ info:
   contact:
     email: tech@interledger.org
 servers:
-  - url: 'https://auth.rafiki.money'
+  - url: 'https://auth.ilp.interledger-test.dev'
```

```suggestion
  - url: 'https://auth.interledger-test.dev'
``` 
`.ilp` not applicable to auth links